<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logging_window</class>
 <widget class="QWidget" name="logging_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Przyłóż kartę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_barman">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Barman</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_waiter">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kelner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
